new/rfc-for-hugs.md

```diff
@@ -0,0 +1,48 @@
+---
+RFC: unassigned
+Author: Aaron Kalin and JJ Asghar
+Status: Draft
+Type: Informational, Process
+---
+
+# Hugging in the Chef Community
+
+In order to bridge the cultural gap between the US and Worldwide community the concept of hugging should be explained.
+
+## Motivation
+
+    As a Chef Community Participant,
+    I want to express positive emotion to or for another Chef Community Participant,
+    so that we can express our friendship physically.
```

Comment:
The "I want to" and "so that" lines seem to be a bit redundant here.  Also, I think it could do more to point out what is core to this RFC which is that we want to do these things *while respecting the boundaries of others*.